db/seeds/custom_contents.rb

```diff
@@ -72,7 +72,7 @@
    placeholders_required: "first-name",
    placeholders_optional: "profile-url"},
   {key: Memberships::LeaveZusatzsektionMailer::CONFIRMATION,
-   placeholders_required: "person-name, sektion-name, terminate-on"},
+   placeholders_required: "person-name, group-name, terminate-on"},
```

Comment:
Are there existing texts in the production DB for these? They would break then, right?